Library/test-helpers/lib.sh

```diff
@@ -1715,6 +1715,254 @@ limeSubmitCommonLogs() {
 
 }
 
+true <<'=cut'
+=pod
+
+=head2 limeconCreateNetwork
+
+Create container network.
+
+    limeconCreateNetwork
+
+=over
+
+=item NETWORK_IP
+
+IP of container network.
+
+=item NAME_NETWORK
+
+Name of container network.
+
+=back
+
+Returns 0.
+
+=cut
+
+limeconCreateNetwork() {
+
+    local NETWORK_IP=$1
+    local NAME_NETWORK=$2
+
+    if [ -z "${NAME_NETWORK}" ] || [ -z "${NETWORK_IP}" ]; then
+        echo "Name of network or network IP is not specified!"
+        return
+    fi
+
+    podman network create --subnet=$NETWORK_IP $NAME_NETWORK
+    echo "Network name: ${NAME_NETWORK} , ip:${NETWORK_IP}"
+
+}
+
+true <<'=cut'
+=pod
+
+=head2 limeconDeleteNetwork
+
+Delete container network.
+
+    limeconDeleteNetwork
+
+=over
+
+=item NAME_NETWORK
+
+Name of the container network.
+
+=back
+
+Returns 0.
+
+=cut
+
+limeconDeleteNetwork() {
+
+    local NAME_NETWORK=$1
+
+    podman network inspect $NAME_NETWORK
+
+}
+
+true <<'=cut'
+=pod
+
+=head2 limeconPrepareAgentImage
+
+Prepare podman image with keylime agent. Add dnf repos into podman image, install keylime agent and
+copy CA cert from the host into image and set needed permissions.
+
+    limeconPrepareAgentImage
+
+=over
+
+=item NAME_TAG
+
+Name of image tag, if paramer empty name tag set as agent_image.
+
+=back
+
+Returns 0.
+
+=cut
+
+limeconPrepareAgentImage() {
+
+    local IMAGE=$1
+    local NAME_TAG=$2
+
+    #copy cacert to work dir for podman build
+    cp /var/lib/keylime/cv_ca/cacert.crt .
```

Comment:
what about implementing optional `--cacert` parameter to your function?

Reply:
I'm not sure if we'll use this function in some cases without cacert...WDYT? You need cacert to properly start the agent in the container.

Reply:
I rather meant making cacert location configurable.

Reply:
Finally I decide to not use --cacert option, but I use parameter -f which specify docker file for build image. If will not be used this parameter, by default will be used Docker file from test-helpers dir. Do you think, that --cacert parameter is still needed?

Reply:
It is not needed... until we find it necessary. I think it's fine not to use it right now.